src/widgets/panel/panel.tsx

```diff
@@ -248,7 +260,17 @@ const panel: PanelWidget = widgetParams => {
         return undefined;
       },
       render(...args) {
-        const [options] = args;
+        const [renderOptions] = args;
+
+        let widgetRenderState;
```

Comment:
We can simplify this branching logic:

```ts
const widgetRenderState = widget.getWidgetRenderState?.(renderOptions) || {};
```

Reply:
https://github.com/algolia/instantsearch.js/pull/4527/commits/c89a118ba4805366bb008fed630b5bc2f290d90f